.github/workflows/test.yml

```diff
@@ -60,10 +60,10 @@ jobs:
         run: cargo test --workspace --exclude builder --doc --verbose && cargo nextest run --workspace --exclude builder --profile ci --verbose -E '!test(tracing_integration_tests::)'
         env:
           RUST_BACKTRACE: 1
-      - name: "[${{ steps.rust-version.outputs.version}}] Tracing integration tests: cargo nextest run --workspace --exclude builder --profile ci --verbose -E 'test(tracing_integration_tests::)'"
+      - name: "[${{ steps.rust-version.outputs.version}}] Tracing integration tests: cargo nextest run --workspace --exclude builder --profile ci --test-threads=1 --verbose -E 'test(tracing_integration_tests::)'"
```

Comment:
/question

would it slow down the CI?

Reply:
This step runs only the integration tests, not the entire test suite. It may add less than 10s to the test job. It will be done long before Miri and the build pipelines on Gitlab.